keyboards/augio/config.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
```suggestion
#define VENDOR_ID       0x6477  // DW 
```
Just to make it unique.